html-tables/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Html Table</title>
+
+    <body>
+      <h1>Student Grade Table</h1>
+      <table>
+        <theading>
+          <tr>
+            <th scope="col">Student Name</th>
+            <th scope="col">Course</th>
+            <th scope="col">Grade</th>
+          </tr>
+        </theading>
+
+        <tbody>
+        <tr>
```

Comment:
Careful about your indentation. This `tr` and the other `tr` elements are children of `tbody`, so they should be indented by one tab.